### PR TITLE
Fix English mistakes in Schema Design

### PR DESCRIPTION
**Pull request by GWphua**
#### Fixed English mistakes in Schema Design Documentation

This PR has:

- [x] been self-reviewed.

### CODE REVIEW

File: docs/ingestion/schema-design.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ two. You won't be able to retrieve the user IDs or compute exact distinct counts
 approximate distinct counts, and you'll reduce your storage footprint.
 
 Sketches reduce memory footprint at query time because they limit the amount of data that needs to be shuffled between
-servers. For example, in a quantile computation, instead of needing to send all data points to a central location
```
Comment (kfaraz) [inline docs/ingestion/schema-design.md:162]: the original text was correct

File: docs/ingestion/schema-design.md [hunk 2]
```diff
@@ -255,7 +255,7 @@ Druid can infer the schema for your data in one of two ways:
 
 You can have Druid infer the schema and types for your data partially or fully by setting `dimensionsSpec.useSchemaDiscovery` to `true` and defining some or no dimensions in the dimensions list. 
 
-When performing type-aware schema discovery, Druid can discover all of the columns of your input data (that aren't in
+When performing type-aware schema discovery, Druid can discover all the columns of your input data (that aren't in
```
Comment (kfaraz) [inline docs/ingestion/schema-design.md:258]:
```suggestion
When performing type-aware schema discovery, Druid can discover all the columns of your input data (that are not present in
```

Comment (GWphua) [reply docs/ingestion/schema-design.md]: done

**Review by kfaraz — APPROVED** [review]
Thanks, @GWphua !

**Pull request merged**